docs/release-notes/9.2021.md

```diff
@@ -0,0 +1,18 @@
+### Improve consumption code 
+_Need to get more details from Zewditu here._ [#1605](https://github.com/dotnet/machinelearning-modelbuilder/issues/1605) [#1567](https://github.com/dotnet/machinelearning-modelbuilder/issues/1567) [#1419](https://github.com/dotnet/machinelearning-modelbuilder/issues/1419)
+
+### Usability 
+- SQL Server Authentication fix [#1540](https://github.com/dotnet/machinelearning-modelbuilder/issues/1540)
+- New Azure Workspace create [#1104](https://github.com/dotnet/machinelearning-modelbuilder/issues/1104), [#1492](https://github.com/dotnet/machinelearning-modelbuilder/issues/1492), [#1654](https://github.com/dotnet/machinelearning-modelbuilder/issues/1654)
+- GPU fix: Multiple versions of CUDA can be installed. Only requirment is Cuda 10.1 is installed and Cuda_Path_v10_1 is set. [#1642](https://github.com/dotnet/machinelearning-modelbuilder/issues/1642)
```

Comment:
We might need to update document as well

Reply:
@luisquintanilla In [install-dependencies](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/install-gpu-model-builder#install-dependencies) section

> Install CUDA v10.1. Make sure you install CUDA v10.1, not any other newer version. You cannot have multiple versions of CUDA installed.

We can remove `You cannot have multiple versions of CUDA installed.`

Reply:
@LittleLittleCloud does this look good?

Reply:
@luisquintanilla LGTM